src/components/parameters.js

```diff
@@ -69,7 +69,8 @@ const useStyles = makeStyles((theme) => ({
         padding: theme.spacing(2),
     },
     grid: {
-        padding: theme.spacing(2),
+        paddingTop: theme.spacing(2),
+        padding: theme.spacing(0),
```

Comment:
Bizarre l'ordre de ce CSS non ? PaddingTop est un sous ensemble de Padding normalement.

Reply:
oui, mais j'ai besoin d'utiliser un paddingTop différent de les autres

Reply:
The applied CSS looks like :
`padding: 0px;
    padding-top: 16px;`
It seems like the most specific property is applied regardless of the order

Reply:
Après tests ça n'a pas d'importance vu que le navigateur prend paddingTop après padding quoi qu'il arrive, mais au niveau de la lecture j'aurai quand même mis padding juste avant paddingTop.

Si on essaye avec un fichier HTML tout simple, on a une différence entre :
`<html>
  <head>
    <title></title>
    <meta content="">
    <style>div {padding-top: 2px; padding:60px; background-color:red }</style>
  </head>
  <body><div> TEST</div></body>
</html>
`
et 

`<html>
  <head>
    <title></title>
    <meta content="">
    <style>div {padding:60px; padding-top: 2px; background-color:red }</style>
  </head>
  <body><div> TEST</div></body>
</html>
`
Dans le premier cas, padding écrase la règle padding-top.